.gitignore

```diff
@@ -2,3 +2,4 @@
 build
 composer.phar
 composer.lock
+/.idea
```

Comment:
I don't think this is needed. You can add it to your global gitignore if you use PHPstorm :)

Reply:
Cool, didn't know about that. Shouldn't have included that either way.